test/cmockatest.c

```diff
@@ -352,6 +353,31 @@ RTR_TEST_END
 RTR_TEST_START(dirfd)
 RTR_TEST_END
 
+/* netdb functions test */
```

Comment:
Can we also have unit tests here? I see that you've added standalone binaries, but they don't really test anything always returning 0.